doespy/doespy/util.py

```diff
@@ -94,7 +94,8 @@ def get_suite_roles_dir():
 
 def get_results_dir():
     prj_dir = get_project_dir()
-    results_dir = os.path.join(prj_dir, "doe-suite-results")
+    results_dir_name = os.environ.get("DOES_RESULTS_DIR", "doe-suite-results")
+    results_dir = os.path.join(prj_dir, results_dir_name)
```

Comment:
Based on the name `DOES_RESULTS_DIR` and the way we e.g., use `DOES_PROJECT_DIR` I would have expected that `DOES_RESULTS_DIR`  is an absolute path rather than a relative path based on the `DOES_PROJECT_DIR`.